pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -5573,3 +5573,8 @@ query B
 SELECT ST_MinimumBoundingCircle(NULL::geometry) IS NULL;
 ----
 true
+
+query T
+SELECT ST_AsText(ST_UnaryUnion(ST_GeomFromText('MULTIPOLYGON(((0 0,4 0,4 4,0 4,0 0),(1 1,2 1,2 2,1 2,1 1)), ((-1 -1,-1 -2,-2 -2,-2 -1,-1 -1)))')));
+----
+MULTIPOLYGON (((-1 -1, -1 -2, -2 -2, -2 -1, -1 -1)), ((0 0, 4 0, 4 4, 0 4, 0 0), (1 1, 2 1, 2 2, 1 2, 1 1)))
```

Comment:
I wanted to test if this function returns an error correctly, but I have not found a geometry argument for which it would throw one.

Reply:
I think invalid Geometries would throw an invalid error.

Reply:
But if I pass invalid Geometry, the error will returned from geometry parser, not `UnaryUnion` function, correct? 🤔

Reply:
not quite, you can create a polygon that the parser likes but is not valid.
see examples here: https://postgis.net/docs/ST_IsValid.html

Reply:
I have tried, but it did not throw any error (PostGIS results):
```
marcinknychala=# select ST_IsValid(ST_GeomFromText('POLYGON((0 0, -1 -1, -1 1, 0 0, 1 1, 1 -1, 0 0))'));
NOTICE:  Ring Self-intersection at or near point 0 0
 st_isvalid
------------
 f
(1 row)

marcinknychala=# SELECT ST_AsText(
  ST_UnaryUnion(ST_GeomFromText('POLYGON((0 0, -1 -1, -1 1, 0 0, 1 1, 1 -1, 0 0))'))
) As output;
                   output
--------------------------------------------
 POLYGON((0 0,-1 -1,-1 1,0 0,1 1,1 -1,0 0))
(1 row)
```
Passing empty geometries does not throw an error either:
```
marcinknychala=# SELECT ST_AsText(
ST_UnaryUnion(ST_GeomFromText('GEOMETRYCOLLECTION(POINT EMPTY, POINT(0 0))'))
) As output;
   output
------------
 POINT(0 0)
(1 row)
```